admin.py

```diff
@@ -107,6 +109,38 @@ class TicketInline(admin.TabularInline):
     extra = 0
 
 
+def process_names_csv(csv_file):
+    """
+    Process a csv file for online orders
+    @param csv_file: A csv file containing names to be set to paid.
+    @return: list of names that had problems being processed and a message summarizing the done process.
+    """
+    names = pandas.read_csv(csv_file, delimiter="\t", header=None)
```

Comment:
At this moment, the CSV should be formatted as "First name" , "Last name". Could we also make a column "Ticket key"?

Reply:
I think that negates the usefulness for the orchestra administrator. If the administrator needs to keep track of the order keys, they needs to look up these keys in the website database, which is slow and one of the main reasons that this feature needs to exist in the first place.